crm_request/models/request_type.py

```diff
@@ -13,3 +13,27 @@ class RequestType(models.Model):
     template_id = fields.Many2one('mail.template',
                                   'Template',
                                   domain="[('model_id', '=', 'crm.claim')]")
+    keywords = fields.Char(
+        string='Keywords',
+        help='List of keywords (separated by a comma ",") who could be '
+             'contained in the demand subject')
+
+    @api.constrains('keywords')
+    def _check_existing_key(self):
+        # Avoid having two identical keys
+        records = self.search([('keywords', '!=', False)])
```

Comment:
You could go from O(n^2) to average O(n) by getting all the keyword in one go, creating a set from the keywords and checking if the length is the same.